sickbeard/logger.py

```diff
@@ -155,6 +155,19 @@ def submit_errors(self):
         if not (sickbeard.GIT_USERNAME and sickbeard.GIT_PASSWORD and len(classes.ErrorViewer.errors) > 0):
             self.log('Please set your GitHub username and password in the config, unable to submit issue ticket to GitHub!')
```

Comment:
Well, needs_update is only set if SR knows it needs an update ... If it doesn't, because checkversion doesn't run for instance, we should complain about that instead ;)

-----Original Message-----
From: "Gaëtan Muller" <notifications@github.com>
Sent: ‎7/‎28/‎2015 8:34 PM
To: "SiCKRAGETV/SickRage" <SickRage@noreply.github.com>
Subject: Re: [SickRage] Check if SR is updated before submitting an issue(#2217)

In sickbeard/logger.py:
> @@ -155,6 +155,19 @@ def submit_errors(self):
>          if not (sickbeard.GIT_USERNAME and sickbeard.GIT_PASSWORD and len(classes.ErrorViewer.errors) > 0):
>              self.log('Please set your GitHub username and password in the config, unable to submit issue ticket to GitHub!')
Isn't there an issue to automatically turning debug on before sending an issue?
—
Reply to this email directly or view it on GitHub.